Content.Client/Chemistry/UI/ReagentDispenserBoundUserInterface.cs

```diff
@@ -32,7 +32,7 @@ protected override void Open()
             // Setup window layout/elements
             _window = this.CreateWindow<ReagentDispenserWindow>();
             _window.Title = EntMan.GetComponent<MetaDataComponent>(Owner).EntityName;
-            _window.HelpGuidebookIds = EntMan.GetComponent<GuideHelpComponent>(Owner).Guides;
```

Comment:
Would be good if you kept `HelpGuidebookIds` as a property on the `FancyWindow` (with a big comment to describe what it's for) which just forwards to the button - that way, people are much more likely to see and use it, and derived classes are a little more implementation-agnostic.